deploy/complete/terraform/schema.yaml.orig

```diff
@@ -0,0 +1,396 @@
+
+source:
+  type: quickstart
```

Comment:
did you plan to include all these other files as well?

Reply:
Should not be included. Fixed.